### ISSUE TITLE
Incorrect HK glyph shape for U+3C18 㰘 and U+5F5D 彝

### ISSUE THREAD

**Issue by tamcy** [adobe-fonts/source-han-sans#345]
![u3c18_5f5d](https://user-images.githubusercontent.com/959433/179675885-5ef0da6a-f7c8-414e-902b-e6ec744759e4.png)

Currently both uses the CN glyph (uni3C18-CN and uni5F5D-CN), but as indicated above, their shapes are different for HK.

**Comment by punchcutter** [comment]
It might be helpful to start making a note of glyphs that are also part of MSCS. Which means HK glyphs like U+3C18 which is part of HKSCS-2008. We need to make room for these while also addressing other issues of incorrect mappings or missing region specific glyphs.

**Comment by tamcy** [comment]
As MSCS is by definition a superset of HKSCS, do it need to state that explicitly for HKSCS characters?

**Comment by punchcutter** [comment]
No, I meant it more as something to keep in mind since a lot of the current issues are about glyph sharing/consolidation, etc.  It's not just about HK glyphs that will also be in the MO set. There are some existing glyphs that are JP, CN, TW, or HK and already can be used for MO, but for some we need a new glyph which in some cases can then also be shared with TW. We already don't have much room for adding a bunch of new glyphs so we need to consolidate and share even more than before.

**Comment by hfhchan** [comment]
The MSCS character set extends Big-5 and HKSCS, but for the IVD characters the base form currently follows Taiwan Conventions.

That said MSCS is a character set standard and not a glyph standard, and no official glyph guideline has been published, so everything is implicit.

**Comment by hfhchan** [comment]
There has been efforts through IRG and the IVD registrar to convince Macao to use the HK conventions instead as the base form, because that is predominantly the form being used in Macao, but that advice was not heeded.

**Comment by tamcy** [comment]
Just spotted another codepoint which exhibits the same nature of issue, so I am adding it here.

In 繭 (U+7E6D), the TW glyph (uni7E6D-TW) is used for HK, but there's a design difference in 糸.

![2023-10-05-U7E6D-Report](https://github.com/adobe-fonts/source-han-sans/assets/959433/e7fb9a92-fb63-410d-9c0e-271718f38afe)

**Comment by tamcy** [comment]
Closing as resolved in v2.005.
